writing-practice/02_review.py

```
@@ -0,0 +1,77 @@
+    else:
+        # Display sentence
+        st.subheader("Original Sentence:")
+        st.write(st.session_state.sentence)
+
+        # Retrieve and display uploaded image
+        if "practice_image" in st.session_state:
+            image = Image.open(io.BytesIO(st.session_state.practice_image))
+            st.image(image, caption="Your handwritten practice", use_container_width=True)
+
+            # Call the grading function with the OCR text and original sentence
+            grading_result = grade_transcription(ocr_text, st.session_state.sentence)
```

Ruff (0.8.2)

Undefined name `ocr_text`

(F821)